lua/copilot/util.lua

```diff
@@ -39,14 +39,4 @@ M.get_completion_params = function()
   return params
 end
 
-M.get_copilot_path = function(plugin_path)
```

Comment:
change this to be:
```
M.get_copilot_path = function()
  local plugin_path = vim.api.nvim_exec("echo expand('<sfile>:p:h:h:h')", true)
  return plugin_path .. "/copilot/index.js"
end
```

Reply:
Indeed this looks better than using the debug function. Thanks!